app/helpers/fines_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Helper module for Fines
+module FinesHelper
+  def nice_status_fee_label(status)
+    return status if status.ends_with?('fee')
+
+    "#{status} fee"
```

Comment:
I thought about just adding this to the model, and would be happy to do that.

Reply:
I'm fine with it here.. seems like it fits the cognitive model of where to place something like this.